pkg/services/blockfetcher/blockfetcher_test.go

```diff
@@ -94,5 +94,6 @@ func TestServiceConstructor(t *testing.T) {
 		}
 		_, err := New(ledger, cfg, logger, mockPut.putBlock, shutdownCallback)
 		require.Error(t, err)
+		require.Contains(t, err.Error(), "no such file or directory")
```

Comment:
"open wallet: open invalid/path/to/wallet.json: The system cannot find the path specified." does not contain "no such file or directory"